prog/vm/github_runner.rb

```diff
@@ -200,6 +182,18 @@ def before_run
     nap 5
   end
 
+  label def create_billing_record
+    BillingRecord.create_with_id(
```

Comment:
I have some thoughts on this, please give me some time to review this PR properly.